src/workforce-management/wwwroot/js/site.js

```diff
@@ -1 +1,29 @@
-﻿// Write your Javascript code.
+﻿$(document).ready(function () {
+
+    /**
+     * Purpose: Checks to see if the input fields within the department are null or !null.
+     * Arguments:
+     *    N/A
+     * Return:
+     *     N/A
+     **/
+    function entryCheck() {
+        if ($(".departmentName").val() && $(".departmentDescription").val()) {
+            $("#departmentBtn").prop("disabled", false)
+        } else {
+            $("#departmentBtn").prop("disabled", true)
+        }
+    }
+
+    $("#departmentBtn").prop("disabled", true);

```

Comment:
rather than disabling this on start, why don't you run the `entryCheck()` on load, that way the button won't be disabled when we work on the edit feature